cmd/sealer/cmd/root.go

```diff
@@ -84,39 +82,87 @@ func init() {
 	rootCmd.AddCommand(cluster.NewClusterCommands()...)
 	rootCmd.AddCommand(image.NewImageCommands()...)
 
-	rootCmd.PersistentFlags().StringVar(&rootOpt.cfgFile, "config", "", "config file of sealer tool (default is $HOME/.sealer.json)")
+	rootCmd.PersistentFlags().StringVar(&rootOpt.cfgFile, "config", "", "config file of sealer tool (default is $HOME/.sealer/config.json)")
```

Comment:
Are there compatibility issues here？

Reply:
no, sealer has no config file up to now.